csrankings-t.csv

```diff
@@ -699,7 +700,7 @@ Tingting Jiang 0001,Peking University,http://www.idm.pku.edu.cn/staff/jiangtingt
 Tingting Yu,University of Cincinnati,https://homepages.uc.edu/~tyu,vcg-v-IAAAAJ
 Tingzhu Huang,Univ. of Maryland - Baltimore County,http://www.csee.umbc.edu/~zt,H7El-ZkAAAAJ
 Tino Weinkauf,KTH Royal Institute of Technology,https://www.kth.se/profile/weinkauf,ZS5iEecAAAAJ
-Tinoosh Mohsenin,Univ. of Maryland - Baltimore County,http://www.csee.umbc.edu/~tinoosh,9grfK9gAAAAJ
+Tinoosh Mohsenin,Johns Hopkins University,https://eehpc.ece.jhu.edu/,9grfK9gAAAAJ

```

Comment:
<img width="1200" alt="Screen Shot 2024-10-26 at 11 26 32 AM" src="https://github.com/user-attachments/assets/617ed9a3-7e11-48bf-94b4-cad1624db070">

From this page: https://www.cs.jhu.edu/people/cs-affiliates/joint-and-secondary-faculty/